docker/.env

```diff
@@ -37,17 +38,17 @@ LP_ETL_FUSEKI_ADMIN_PASSWORD=admin
 LP_ETL_FUSEKI_JVM_ARGS=-Xmx2g
 
 RDF_FINGERPRINTER_FUSEKI_DATA_FOLDER=fuseki-fingerprinter-volume
-RDF_FINGERPRINTER_FUSEKI_PORT=3020
-RDF_FINGERPRINTER_FUSEKI_LOCATION=http://fuseki-fingerprinter
+RDF_FINGERPRINTER_FUSEKI_PORT=3030
```

Comment:
In the final solution, this port shall still be 3020; but only after the config.py files would have been changed.

Reply:
that was done, I see; that is great, I suggest the change inline then.